ingest-asset-opex-creator/src/test/scala/uk/gov/nationalarchives/ingestassetopexcreator/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -155,6 +155,9 @@ class ExternalServicesTestUtils(dynamoServer: WireMockServer, s3Server: WireMock
        |      "sortOrder": {
        |        "N": "1"
        |      },
+       |      "childCount": {
+       |        "N": "0"
```

Comment:
Just out of curiosity, is it possible to have 0 children? If not, should we do a check/prevent this from happening?

Reply:
It's possible to have zero children. Files don't have any children.